infection_monkey/monkey.py

```diff
@@ -70,7 +73,16 @@ def start(self):
 
         if firewall.is_enabled():
```

Comment:
If we're already touching main, lets move 
`        if firewall.is_enabled():`
`            firewall.add_firewall_rule()`

To right next to `create_control_tunnel`, so we don't add a firewall rule we won't need.

Reply:
fixed